wagtail/admin/views/account.py

```diff
@@ -144,6 +144,8 @@ class NameEmailSettingsPanel(BaseSettingsPanel):
 
     @cached_property
     def title(self):
+        from wagtail.admin.views.account import email_management_enabled
+
```

Comment:
This is unrelated, please rebase your branch and ensure you don't have unrelated commits.

Reply:
Okay, I'll do that along with my next commit. Thank you sir!

Reply:
Hello sir, Should I create a new pr, as everything I do now stacks on top of this one

Reply:
As per other comments, you will either need to create a new branch that has the changes only for this issue or revert / make the changes on this branch that is only for the one issue.